templates/default/layout/html/layout.erb

```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
     <%= erb(:headers) %>
+    <meta name="description" content="The main documentation for the openhab-scripting helper library.">
```

Comment:
this layout is rendered for each class/module or markdown file.

wouldn't this be more useful to actually use the short description of the class, or the first paragraph of the markdown file?

Reply:
OK, I _think_ this should do it. Using a `@description` (custom) tag in the additional markdown files, otherwise use the docstring which seems to be the first line of the top Overview section.